app/controllers/works_controller.rb

```diff
@@ -455,6 +455,7 @@ def update
   end
 
   def update_tags
+    set_work_tag_error_messages
```

Comment:
Thank you -- good catch! Taking the Ready to Merge label off and putting Needs Coder Action on.